portal/portal-tool/tool/src/java/org/sakaiproject/portal/tool/ToolPortal.java

```diff
@@ -265,71 +264,4 @@ protected void doPost(HttpServletRequest req, HttpServletResponse res)
 		doGet(req, res);
 	}
 
-	/**
-	 * Setup the request attributes with information used by the tools in their
-	 * response.
-	 * 
-	 * @param req
-	 * @param res
-	 * @param p
-	 * @param skin
-	 * @throws ToolException
-	 */
-	// NOTE: This code is duplicated in SkinnableCharonPortal.java
-	// make sure to change code both places
-	protected void setupForward(HttpServletRequest req, HttpServletResponse res,
```

Comment:
Based on my testing, it doesn't need to be duplicated here. Tell me if I'm missing something.